From 778d1146e69e414440e9708ebc02ca741c6ddbfc Mon Sep 17 00:00:00 2001
From: Maurice Kalinowski <maurice.kalinowski@qt.io>
Date: Tue, 30 Oct 2018 09:34:56 +0100
Subject: [PATCH 1/1] Disable x86 check on declarative

---
 recipes-qt/qt5/qtdeclarative_git.bb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/recipes-qt/qt5/qtdeclarative_git.bb b/recipes-qt/qt5/qtdeclarative_git.bb
index db41649..f95ca81 100644
--- a/recipes-qt/qt5/qtdeclarative_git.bb
+++ b/recipes-qt/qt5/qtdeclarative_git.bb
@@ -24,6 +24,7 @@ do_configure_prepend() {
     # disable qtxmlpatterns test if it isn't enabled by PACKAGECONFIG
     sed -e 's/^\(qtHaveModule(xmlpatterns)\)/OE_QTXMLPATTERNS_ENABLED:\1/' -i ${S}/src/imports/imports.pro
     sed -e 's/^\(!qtHaveModule(xmlpatterns)\)/!OE_QTXMLPATTERNS_ENABLED|\1/' -i ${S}/tests/auto/quick/quick.pro
+    sed -e 's/#ifdef Q_PROCESSOR_X86_32/#if 0/g' -i ${S}/src/qml/qml/v8/qv8engine.cpp
 }
 
 do_install_append_class-nativesdk() {
-- 
2.17.1

